src/python/pants/core/util_rules/lockfile_metadata.py

```diff
@@ -16,6 +16,7 @@
 
 
 class LockfileScope(Enum):
+    JVM = "jvm"
     PYTHON = "python"
```

Comment:
@Eric-Arellano : Not a blocker for this change, but this seems like something that should be computed from one of the `@union`s involved here?

Reply:
Maybe which metadata class to create when generating a lockfile, but nothing else.